Franciscoginzo encoder

fginzo opened the pull request:
- creadas las clases CExerciseEncoding y CEncoderJeringozo
- creadas las clases CExerciseEncodingTest y CEncoderJeringozoTest
- creada la clase CExercise
- renombradas las clases FranciscoGinzo, FranciscoGinzoTest, ExerciseGitSetUp, ExerciseStudentCreation y ExerciseStudentCreationTest
- extendidas las clases CExerciseEncoding, CExerciseGitSetUp y CExerciseStudentCreation de CExercise

fginzo committed:
+ creada las clases CExerciseEncoding y CEncoderJeringozo

fginzo committed:
+ creada las clases CExerciseEncoding y CEncoderJeringozo

fginzo committed:
+ creada las clases CExerciseEncoding y CEncoderJeringozo

fginzo committed:
+ creadas las clases CExerciseEncoding y CEncoderJeringozo
+ creada la clase abstracta CExercise
+ renombradas las clases FranciscoGinzo, FranciscoGinzoTest, ExerciseGitSetUp, ExerciseStudentCreation y ExerciseStudentCreationTest
+ extendidas las clases CExerciseEncoding, CExerciseGitSetUp y CExerciseStudentCreation de CExercise

fginzo committed:
+ creadas las clases CExerciseEncoding y CEncoderJeringozo
+ creada la clase abstracta CExercise
+ renombradas las clases FranciscoGinzo, FranciscoGinzoTest, ExerciseGitSetUp, ExerciseStudentCreation y ExerciseStudentCreationTest
+ extendidas las clases CExerciseEncoding, CExerciseGitSetUp y CExerciseStudentCreation de CExercise
+ creada las clases CExerciseEncodingTest y CEncoderJeringozoTest

fginzo committed:
+ creadas las clases CExerciseEncoding y CEncoderJeringozo
+ creada la clase abstracta CExercise
+ renombradas las clases FranciscoGinzo, FranciscoGinzoTest, ExerciseGitSetUp, ExerciseStudentCreation y ExerciseStudentCreationTest
+ extendidas las clases CExerciseEncoding, CExerciseGitSetUp y CExerciseStudentCreation de CExercise
+ creadas las clases CExerciseEncodingTest y CEncoderJeringozoTest

didierrenardub commented:
Francisco,

Vi muuuuy por encima el ejercicio y parecía bastante bien. Lo que te voy a pedir antes de aceptar el pull request es que mergees con el master (que hubo cambios de parte de Diego) y restaures CExercisesMain y CStudentStatistics a como estaba, porque vi que comentaste cosas al respecto que no debían ser comentadas.

Saludos,
Didier

fginzo committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into franciscoginzo_encoder

# Conflicts:
#	ejercicios/src/ar/edu/ub/pcsw2016/diegovega/Diego.java
#	ejercicios/src/ar/edu/ub/pcsw2016/diegovega/GitSetup.java
#	ejercicios/src/ar/edu/ub/pcsw2016/diegovega/StudentCreation.java

fginzo commented:
hola Didier, por GitHub desktop hice el merge de los cambios en los archivos de DSV. creería que lo hice bien porque ahora en GH leo que no hay conflictos con la base branch. entiendo que también se restauraron los archivos originales del Main y Statistics. si no fuera así, avisame cómo tendría que hacerlo. igual por cualquier otra cosa que me falte.
saludos, Francisco

didierrenardub commented:
Hola Francisco,

Vi que se hicieron los cambios en el main pero no en statistics. Al menos ingresando acá:
https://github.com/didierrenardub/ub_pcsw_2016/blob/franciscoginzo_encoder/ejercicios/src/ar/edu/ub/pcsw2016/CStudentStatistics.java

Asumo que te faltó un push al respecto.

Saludos,
Didier

fginzo commented:
hola Didier, qué raro. acabo de hacer un Update Project - Merge en IntelliJ y sólo saltaron los cambios de DVS. resolví el conflicto aceptando los de él y después hice el merge. por separado me fijé en GH desktop y me dice que estoy al día con el master. y acá leo que no hay conflictos entre mi branch y la base branch.  qué me sugerís hacer?

didierrenardub commented:
Andá a CStudentStatistics, descomentá el código comentado perteneciente a Diego, add, commit, push y listo!

Una vez hecho eso, reviso bien en detalle el código de tu ejercicio y con eso cerramos, en caso de que no haya ningún problema.

Saludos,
Didier

fginzo committed:
+ en CStudentStatistics:
    1. descomenté el código relativo a DSV
    2. renombré las clases a las nuevas creadas por DSV

fginzo commented:
listo!
